sherpa/optmethods/tests/test_optmethods.py

```diff
@@ -39,13 +38,22 @@ def tst_all( self, name, fct, fmin, x0, xmin, xmax,
                  iprint=False ):
         self.tst( optfcts.neldermead, name + self.nm, fct, fmin,
                   x0, xmin, xmax, iprint=iprint )
-##         self.tst( stogo.stogo, name + '_stogo', fct, fmin,
-##                   x0, xmin, xmax, iprint=iprint )
         self.tst( optfcts.montecarlo,  name + self.mc , fct, fmin,
                   x0, xmin, xmax, iprint=iprint, maxfev=8192*len(x0) )
         self.tst( optfcts.lmdif, name + self.lm, fct, fmin,
                   x0, xmin, xmax, iprint=iprint )
 
+        #
+        # If the following lines are uncommented out, we get errors
+        # when running the optimization test
+        #
+        # try:
+        #     import iminuit
+        #     self.tst(optfcts.midnight, name + '_minuit', fct, fmin,
+        #              x0, xmin, xmax, iprint=iprint)
+        # except ImportError:
+        #     pass
+
```

Comment:
Rather than protecting the test in a try/except block, we should use the `unittest.skipIf` decorator (or one of the wrappers around such as `sherpa.utils.requires_package`), since then we have better tracking of the total number of tests. @olaurino can (hopefully) provide info on the preferred syntax here.